avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -2024,3 +2076,120 @@ cpdef double SamosATfric(double rho, double Rs0, double mu, double kappa, double
   div = math.log(div) / kappa + B
   cdef double tau = p * mu * (1.0 + Rs0 / (Rs0 + Rs)) + rho * v * v / (div * div)
   return tau
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+@cython.cdivision(True)
+def computeIniMovement(cfg, particles, dem, dT, fields):
+  """ add artifical viscosity effect on velocity for splitIniValueToArraySteps
```

Comment:
not sure I understand what you mean here

Reply:
sentence should end after velocity, changed this